man/coap-rd.txt.in

```diff
@@ -45,6 +92,13 @@ coap-rd -A ::1
 ----
 Let the server listen on localhost (port 5683).
 
+* Example
+----
+coap-server -A ::1 -k mysecretKey -h myhint
```

Comment:
This would be "coap-rd" and not "coap-server".

Reply:
Corrected and code pushed